instrumentation/opentelemetry-instrumentation-celery/tests/test_tasks.py

```diff
@@ -92,6 +92,42 @@ def test_task(self):
         self.assertEqual(consumer.parent.span_id, producer.context.span_id)
         self.assertEqual(consumer.context.trace_id, producer.context.trace_id)
 
+    def test_queue_name(self):
```

Comment:
Since this is a constanct I would just add the attribute to the already available tests `test_task` and `test_task_raises`.

Reply:
done